app/src/main/java/org/xwiki/android/authenticator/exceptions/ReadPhoneStateException.java

```diff
@@ -0,0 +1,18 @@
+package org.xwiki.android.authenticator.exceptions;
```

Comment:
You should add the XWiki header to new files (see other java files). I indeed noticed it's missing in other older files, well need to fix it to be clean.

Reply:
Got it

Reply:
Thanks for also fixing other files :)